### PR TITLE
feat: v35

### PR DESCRIPTION
**Pull request by limpbrains**
- new layout for NavigationHeader to allow 2 buttons on right and how it work nicely with long titles
- new Beta warnings
- new No Multidevince warnings
- new screen for Profile details
- HighBalance warning level decreased to 100 USD
- fix e2e tests for all of that

Screenshot 2023-03-15 at 14 10 46

### CODE REVIEW

File: src/screens/Settings/Backup/MultipleDevices.tsx [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+import { useTranslation } from 'react-i18next';
+
+import { Text01S } from '../../../styles/text';
+import GradientView from '../../../components/GradientView';
+import BottomSheetNavigationHeader from '../../../components/BottomSheetNavigationHeader';
+import GlowImage from '../../../components/GlowImage';
+import Button from '../../../components/Button';
+import type { BackupScreenProps } from '../../../navigation/types';
+
+const imageSrc = require('../../../assets/illustrations/exclamation-mark.png');
```
Comment (pwltr) [inline src/screens/Settings/Backup/MultipleDevices.tsx]:
```suggestion
const imageSrc = require('../../../assets/illustrations/phone.png');
```

Comment (pwltr) [reply src/screens/Settings/Backup/MultipleDevices.tsx]: Also, text for `mnemonic_data_text` has been updated.

File: src/screens/Profile/ProfileDetails.tsx [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+				</View>
+				<View style={styles.buttonContainer}>
+					<Button
+						style={styles.button}
+						text={t('profile_edit')}
+						size="large"
+						onPress={(): void => {
+							navigation.navigate('ProfileEdit');
+						}}
+					/>
```
Comment (pwltr) [inline src/screens/Profile/ProfileDetails.tsx:65]: Missing horizontal margin on this button

**Review by pwltr — CHANGES_REQUESTED** [review]

File: src/screens/Profile/ProfileDetails.tsx (hunk 2 above)
Comment (limpbrains) [reply src/screens/Profile/ProfileDetails.tsx:65]: fixed

File: src/screens/Settings/Backup/MultipleDevices.tsx (hunk 1 above)
Comment (limpbrains) [reply src/screens/Settings/Backup/MultipleDevices.tsx]: fixed

**Review by pwltr — APPROVED** [review]

**Pull request merged**